### DISCUSSION TITLE
Upload logs?

### DISCUSSION THREAD

**Discussion by robwaltersundaysky** [jetstreamapp/jetstream#1021] [Q&A]
Does Jetstream keep an internal log of uploads, such as object record loads? I'm looking for a way to check what I uploaded on a specific day.

**Comment by paustint** [comment]
@robwaltersundaysky - Jetstream does not store any record data of any kind on it's servers and our server logs would only show that your user initiated a load, but wouldn't be able to know anything other than that.

Do you recall if you the Batch or Bulk API?

If you used the bulk API then you may be able to look at the `Bulk Data Load Jobs` in the Salesforce setup menu to get access to the data that was loaded - but this only goes back 7 days.
